webui/react/src/components/ColumnPickerMenu.test.tsx

```diff
@@ -0,0 +1,123 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { DefaultTheme, UIProvider } from 'hew/Theme';
+import { Loaded } from 'hew/utils/loadable';
+import _ from 'lodash';
+
+import { V1LocationType } from 'services/api-ts-sdk';
+import { ProjectColumn } from 'types';
+
+import ColumnPickerMenu, { locationLabelMap } from './ColumnPickerMenu';
+import { initialVisibleColumns, projectColumns } from './ColumnPickerMenu.test.mock';
+import { ThemeProvider } from './ThemeProvider';
+
+const locations = [
+  V1LocationType.EXPERIMENT,
+  [V1LocationType.VALIDATIONS, V1LocationType.TRAINING, V1LocationType.CUSTOMMETRIC],
+  V1LocationType.HYPERPARAMETERS,
+];
+
+const setup = (initCols?: string[]) => {
+  const onVisibleColumnChange = vi.fn();
+  render(
+    <UIProvider theme={DefaultTheme.Light}>
+      <ThemeProvider>
+        <ColumnPickerMenu
+          initialVisibleColumns={initCols ?? initialVisibleColumns}
+          pinnedColumnsCount={0}
+          projectColumns={Loaded(projectColumns as ProjectColumn[])}
+          projectId={1}
+          tabs={locations}
+          onVisibleColumnChange={onVisibleColumnChange}
+        />
+      </ThemeProvider>
+    </UIProvider>,
+  );
+  return {
+    onVisibleColumnChange,
+  };
+};
+
+describe('ColumnPickerMenu', () => {
+  it('should deselect columns', async () => {
+    const { onVisibleColumnChange } = setup();
+    fireEvent.click(await screen.findByRole('button'));
```

Comment:
can we use `userEvent` instead of `fireEvent`?
https://testing-library.com/docs/user-event/intro/#difference-to-fireevent